code/render-code.sh

```diff
@@ -22,46 +22,47 @@ echo Expecting \"$EXPECT\" with {\"$PARAMNAME\": \"$PARAMVALUE\"} returning \"$R
 #echo "----"
 #echo ${PARAMNAME} ${PARAMVALUE} ${RESULTCOLUMN}
 
-echo "Adding language examples to $TARGET, Hit ctrl-c to abort"
+echo "Adding language examples to $TARGET, hit <CTRL-c> to abort, <ENTER> to proceed"
 read
 
 mkdir -p $TARGET/code
 #cp -r code $TARGET
+pushd code
+    Q=`/bin/echo -n "$QUERY" | tr '\n' '§'`
 
-Q=`/bin/echo -n "$QUERY" | tr '\n' '§'`
+    echo $QUERY
+    lines=`echo $QUERY | wc -l`
+    echo Lines $lines
+    echo $Q
 
-echo $QUERY
-lines=`echo $QUERY | wc -l`
-echo Lines $lines
-echo $Q
+    URL="neo4j+s:\/\/demo.neo4jlabs.com:7687"
+    BOLTURL="bolt:\/\/<HOST>:<BOLTPORT>"
 
-URL="neo4j+s:\/\/demo.neo4jlabs.com:7687"
-BOLTURL="bolt:\/\/<HOST>:<BOLTPORT>"
+    for file in `ls */?xample.*`; do
+        LANG=${file%%/*}
+        echo "Updating $file"
+        mkdir -p $TARGET/code/$LANG
+        cp $file $TARGET/code/$file
+        indent=`grep 'MATCH (m:Movie' $TARGET/code/$file | cut -d'M' -f1 | cut -d'"' -f1`
+    #    echo "Indent #$indent#"
+        if [ $LANG == "java" ]; then
+          Q2=`/bin/echo -n "$QUERY" | sed -e "s/\(.*\)/$indent\"\1\" +/g" | tr '\n' '§' | sed -e 's/\+§$/;§/g'`
+        else
+          Q2=`/bin/echo -n "$QUERY" | sed -e "s/\(.*\)/$indent\1/g" | tr '\n' '§'`
+        fi
+    #   echo "$Q2"
+        sed -i.tmp -e "s/^.*MATCH (m:Movie.*$/$Q2/g" $TARGET/code/$file
```

Comment:
on osx I think you can also use `-i""` to not save any backups but not sure, need to google it :)

Reply:
https://stackoverflow.com/a/2420579/728812

Reply:
`-i ""` for Mac OS
`-i""` for Linux :(
will do an explicit removal instead